src/applications/hca/containers/App.jsx

```diff
@@ -71,24 +61,16 @@ const App = props => {
         'view:totalDisabilityRating': parseInt(totalDisabilityRating, 10) || 0,
       };
 
-      if (hasSavedForm || typeof hasSavedForm === 'undefined') {
-        setFormData({
-          ...formData,
-          ...defaultViewFields,
-          'view:userDob': parseVeteranDob(user.dob),
-        });
-      } else if (isLoggedIn) {
+      if (hasSavedForm || typeof hasSavedForm === 'undefined' || isLoggedIn) {
```

Comment:
We don't need the `hasSavedForm` or the `typeof` conditional checks here. Those only applied to the SIP forms. All that needs checked here is the authentication status.